src/error.js

```diff
@@ -544,6 +544,9 @@ export function getErrorReportData(
   if (IS_ESM) {
     runtime = 'esm';
     data['esm'] = '1';
+  } else if (IS_SXG) {
```

Comment:
I think `IS_SXG` also implies `IS_ESM`, so this has to go first.

Reply:
Done.

Reply:
great catch. thanks @jridgewell